Building 1.8.7-p375 fails due to the absence of SVN repos

1.8.7-p375 is supposed to be fetched from the svn.ruby-lang.org:
  https://github.com/rbenv/ruby-build/blob/master/share/ruby-build/1.8.7-p375
However, svn.ruby-lang.org seems to be closed.

1.8.7-p374 is successfull because it has the tarball at ruby-lang.org.

### Steps to reproduce the behavior
```
LANG=C rbenv install --verbose 1.8.7-p375
```
### Expected vs. actual behavior
1.8.7-p375 is supposed to be fetched from the svn.ruby-lang.org, but failed.

### Logs
```
/tmp/ruby-build.20221228133048.1919217.vQ8zKM ~/prj/private/nenga/2023
Checking out http://svn.ruby-lang.org/repos/ruby/branches/ruby_1_8_7...
Redirecting to URL 'https://git.ruby-lang.org/repos/ruby/branches/ruby_1_8_7':
svn: E170013: Unable to connect to a repository at URL 'https://git.ruby-lang.org/repos/ruby/branches/ruby_1_8_7'
svn: E160013: '/repos/ruby/branches/ruby_1_8_7' path not found

BUILD FAILED (Ubuntu 20.04 using ruby-build 20221225-4-gfa46d54)
```
<!-- Paste the output from your terminal. Please enclose the pasted content within triple backticks. If the output suggests the full log was written to a file, copy the contents of that file to a new Gist and link it here. -->

--- a/share/ruby-build/1.8.7-p375
+++ b/share/ruby-build/1.8.7-p375
@@ -1,3 +1,3 @@
 require_gcc
-install_svn "ruby-1.8.7-p375" "http://svn.ruby-lang.org/repos/ruby/branches/ruby_1_8_7" "44351" warn_eol autoconf auto_tcltk standard
+install_git "ruby-1.8.7-p375" "https://github.com/ruby/ruby.git" "dabc90361d1b71fcac4f606eb1eb1fa25c047fd9" warn_eol autoconf auto_tcltk standard
 install_package "rubygems-1.6.2" "https://rubygems.org/rubygems/rubygems-1.6.2.tgz#cb5261818b931b5ea2cb54bc1d583c47823543fcf9682f0d6298849091c1cea7" ruby

